main/course_info/delete_course.php

```diff
@@ -25,29 +25,49 @@
 }
 
 $tool_name = get_lang('DelCourse');
-
-if (isset($_GET['delete']) && $_GET['delete'] === 'yes') {
-    CourseManager::delete_course($_course['sysCode']);
-
-    // DELETE CONFIRMATION MESSAGE
-    Session::erase('_cid');
-    Session::erase('_real_cid');
-    $message = '<h2>'.get_lang('Course').' : '.$current_course_name.' ('.$current_course_code.') </h2>';
-    $message .= get_lang('HasDel');
-    $message .= '<br /><br /><a href="../../index.php">'.get_lang('BackHome').'</a>';
+$type_info_message = 'warning';
+if (isset($_GET['delete']) && $_GET['delete'] === 'yes' && $_GET['course_code'] && !empty($_GET['course_code'])) {
+    if ($current_course_code == $_GET['course_code']) {
+        CourseManager::delete_course($_course['sysCode']);
+        // DELETE CONFIRMATION MESSAGE
+        Session::erase('_cid');
+        Session::erase('_real_cid');
+        $message = '<h2>' . get_lang('Course') . ' : ' . $current_course_name . ' (' . $current_course_code . ') </h2>';
+        $message .= get_lang('HasDel');
+        $message .= '<br /><br /><a href="../../index.php">' . get_lang('BackHome') . '</a>';
+    } else {
+        /* message if code course is incorrect */
+        $message = '<h2>' . get_lang('Course') . ' : ' . $current_course_name . ' (' . $current_course_code . ') </h2>';
+        $message .= '<p>'.get_lang('CourseRegistrationCodeIncorrect').'</p>';
+        $message .= '<p><a class="btn btn-primary" href="' . api_get_path(WEB_CODE_PATH) . 'course_info/delete_course.php?' . api_get_cidreq() . '">' . get_lang('BackToPreviousPage') . '</a>';
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 192 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5721344)